app.rb

```diff
@@ -0,0 +1,11 @@
+require "sinatra"
+require "sinatra/reloader"
+
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postrgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+
+require_relative "db/connection"
+require_relative "../models/tenant"
```

Comment:
Take a look at your file structure -- you don't have to go up a level in these paths for the models. That's why your `app.rb` is erroring out. Remember that your Terminal errors in Sinatra give you line numbers for debugging...